pybop/optimisers/scipy_optimisers.py

```diff
@@ -1,85 +1,193 @@
+import warnings
+
 import numpy as np
 from scipy.optimize import differential_evolution, minimize
 
-from .base_optimiser import BaseOptimiser
+from pybop import Optimisation
 
+DEFAULT_SCIPY_MINIMIZE_OPTIONS = dict(
+    method="Nelder-Mead",
+    jac=False,
+    tol=1e-5,
```

Comment:
If we rename this variable to `threshold` the SciPy and Pints' classes will be able to use the same kwarg.

Reply:
True, but keeping this keyword will help users who refer to the SciPy documentation. We could instead make `tol` and `threshold` synonyms (like `maxiter` and `max_iterations`) in a future PR.